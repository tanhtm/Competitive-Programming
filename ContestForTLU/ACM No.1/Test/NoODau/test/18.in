278 96
2 2 15 19 25 32 38 46 48 51 54 56 63 67 69 71 73 73 74 75 76 79 81 86 86 98 99 99 102 104 114 116 119 119 122 123 123 125 126 129 131 134 135 138 141 145 151 153 154 155 157 157 161 162 163 163 168 168 175 187 188 192 194 197 198 204 206 213 215 221 228 236 238 239 245 250 255 258 259 263 273 274 282 282 289 292 293 295 296 296 298 301 303 306 316 319 319 324 328 330 335 335 339 339 341 351 353 356 357 359 361 363 363 369 370 370 375 380 389 389 395 400 402 405 410 415 416 416 424 430 433 437 438 443 448 455 478 482 485 487 489 491 492 497 498 500 501 502 512 515 515 519 523 526 528 529 534 539 544 545 550 552 561 577 578 578 585 587 588 604 604 622 624 629 629 630 632 633 634 640 641 644 646 652 653 657 664 667 670 670 678 683 686 687 692 704 705 706 706 709 718 719 720 725 731 731 732 738 740 754 754 758 762 769 772 774 776 784 786 789 792 795 797 798 804 807 807 818 821 829 840 851 851 853 853 859 859 862 867 872 873 873 875 880 881 885 886 887 887 891 895 905 908 911 912 915 919 921 944 955 960 963 969 973 976 977 977 977 981 983 985 985 986 986 990 992 997 997 
127 184 720 676 257 956 948 997 837 597 29 822 99 99 975 498 838 286 851 280 952 497 147 955 571 597 105 601 927 45 919 405 228 990 80 837 946 28 833 134 976 213 307 74 311 633 923 148 270 774 780 574 270 278 528 840 874 985 792 152 29 710 557 608 52 988 444 349 367 629 834 343 841 140 768 504 773 691 651 42 464 782 615 85 59 495 924 285 479 716 436 859 777 344 466 180 
