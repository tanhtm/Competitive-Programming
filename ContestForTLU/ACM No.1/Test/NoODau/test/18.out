40
60
203
191
78
261
260
277
231
170
6
230
27
27
265
145
231
85
232
83
260
144
47
260
164
170
31
170
259
8
257
124
71
275
23
231
260
6
231
42
265
68
95
19
95
178
259
47
81
216
218
164
81
83
155
231
243
271
221
48
6
201
163
172
11
275
135
106
114
174
231
106
232
45
214
149
216
195
184
8
137
218
172
24
13
144
259
85
138
201
132
236
218
106
137
60
